src/injections/withPostFailure/README.md

```diff
@@ -0,0 +1,32 @@
+## withPostFailure
+
+The `withPostFailure` injector allows to inject behaviour after the service call fails and the `onFailure` effect is executed. This means that the `${action.target}Loading` and `${action.target}Error` are set before `withPostFailure` is called.
+
+Example:
+
+```
+  import { withPostFailure } from 'redux-recompose';
+
+  const actionCreators = {
+    someAction: data => ({
+      type: actionType,
+      target: someTarget,
+      service: someService,
+      payload: data,
+      injections: [
+        withPostFailure((dispatch, response, state) => {
+          /* insert here whatever logic
+           * you want to execute after a failed service call.
+           * This is particularly userful to dispatch side efects for errored service calls.
```

Comment:
I don't like the word `errored` :stuck_out_tongue_closed_eyes: . Maybe `failed`?

It would be cool to add somehere here about showing error messages or redirecting to another screen if you want to.

Reply:
I agree